src/network/core_utils.rs

```diff
@@ -887,4 +889,18 @@ impl CoreUtils {
         let response = &hash_string[0..length];
         response.to_string()
     }
+
+    // get a sysctl value by the value's namespace
+    pub fn lookup_sysctl_value(ns_value: &str) -> Result<String, SysctlError> {
```

Comment:
I thought all the `sysctl` utils were in firewall crate. But this is also fine.

Reply:
i was asked to move it